androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/common/Logo.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import co.nimblehq.avishek.phong.kmmic.android.R
+
+@Composable
+fun Logo(modifier: Modifier = Modifier) {
```

Comment:
Same here.

Reply:
Updated in 09abeb08dec08b3959cd0e8f2837267c041bde8b